test/extended/threads/atomic_with_offsets.wast

```diff
@@ -0,0 +1,114 @@
+;; atomic operations win non-zero offsets
+
+(module
+  (memory 1 1 shared)
+
+  (func (export "initOffset") (param $value i64) (param $offset i32) (i64.store offset=33 (local.get $offset) (local.get $value)))
```

Comment:
Much better, but could we add some variations to the offset not just 33? Some aligned, some unaligned, and some bigger values.